resilience4j-ratpack/src/main/java/io/github/resilience4j/ratpack/bulkhead/BulkheadConfigurationProperties.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2019 Dan Maas
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.github.resilience4j.ratpack.bulkhead;
+
+import com.google.common.base.Strings;
+import io.github.resilience4j.core.ConfigurationNotFoundException;
+import io.github.resilience4j.core.lang.Nullable;
+
+import javax.validation.constraints.Min;
+import java.util.HashMap;
+import java.util.Map;
+
+public class BulkheadConfigurationProperties {
```

Comment:
Can we share the code between Ratpack and Spring Boot and move it into resilience4j-framework-common?

Reply:
Same